packages/lib-webpack/src/webpack/PatchContainerEntryPlugin.ts

```diff
@@ -2,7 +2,7 @@ import { REMOTE_ENTRY_SCRIPT, REMOTE_ENTRY_CALLBACK } from '@openshift/dynamic-p
 import * as webpack from 'webpack';
 
 export class PatchContainerEntryPlugin implements webpack.WebpackPluginInstance {
-  constructor(private readonly pluginName: string) {}
+  constructor(private readonly pluginName: string, private readonly pluginVersion: string) {}
```

Comment:
Did we want this to be optional? `= ''` or something? Seems like you just upped the requirement to use it.